mitmproxy/contentviews/javascript.py

```diff
@@ -12,9 +54,17 @@ class ViewJavaScript(base.View):
         "text/javascript"
     ]
 
+    def format_text(self, text):
+        for line in text:
+            yield [("text", line)]
+
     def __call__(self, data, **metadata):
-        opts = jsbeautifier.default_options()
-        opts.indent_size = 2
         data = data.decode("utf-8", "replace")
-        res = jsbeautifier.beautify(data, opts)
-        return "JavaScript", base.format_text(res)
+        if len(data) < 1024:
+            opts = jsbeautifier.default_options()
```

Comment:
We should strive to remove jsbeautifier entirely here, that's one dependency less to worry about. :)
(I know we still use it for CSS; should ultimately be removed there as well)